### PR TITLE
Slight formatting changes in README

### PR DESCRIPTION
**Pull request by harsh183**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,31 +20,24 @@ Welcome to the Skulpt developer community! Check out the ideas list below. And t
...
 
-5. Expand and clean up the foreign function API.  This API is critical for implementing parts of the standard library.
+3. Expand and clean up the foreign function API.  This API is critical for implementing parts of the standard library.
 
+4. Currently builtin types (list, tuple, string, etc) are not subclassable.  Making the builtins subclassable would eliminate several known bugs in Skulpt.
 
-1. Currently builtin types (list, tuple, string, etc) are not subclassable.  Making the builtins subclassable would eliminate several known bugs in Skulpt.
+5. Implement decorators.  This would enable a whole bunch of pure python modules to be added to skulpt, plus allow us to add things like properties in a nice way.
```
Comment (ebertmi) [inline README.md]: Actually, skulpt already has some basic decorator support. See https://github.com/skulpt/skulpt/pull/593.

@albertjan Are there any limitations, e.g. multiple decorators not supported?

Comment (harsh183) [reply README.md]: Just confirm? I'll change that for sure.

Comment (albertjan) [reply README.md]: I have no idea I've never tried to decorators 😄 one definitely works. and with #736 this more and more of those use cases will work.

**Comment by harsh183** [comment]
Does this work out better?

**Pull request merged**